release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -2347,6 +2347,46 @@ Identify the disk that holds the live image, in this example `nvme0n1p3`, and ru
 
 * There is currently a known issue with OVN-Kubernetes that whenever the `NetworkManager` service restarts the node will lose network connectivity and must be recovered. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2074009[*BZ#2074009*])
 
+* If you have {sandboxed-containers-first}, when upgrading a cluster, you might encounter an issue where the Machine Config Operator (MCO) pod changes to a `CrashLoopBackOff` state and the `openshift.io/scc` annotation of the pod displays `sandboxed-containers-operator-scc` instead of the default `hostmount-anyuid` value.
++
+If this happens, temporarily change the `seLinuxOptions` strategy in the `sandboxed-containers-operator-scc` SCC to the less restrictive `RunAsAny`, so that the admission process does not prefer it over the `hostmount-anyuid` SCC.
```

Comment:
Do we typically include procedural steps in the "Known issues" section of the Release Notes? You could consider moving the procedure to a module in a troubleshooting page somewhere and then linking to that from the Release Notes.

Reply:
@pneedle-rh - this known issue is only for this release and is only an issue until users have upgraded to sandboxed containers 1.3. We generally have separate release notes, but product experience required we add this known issue to the OCP 4.11 notes, and it will be removed for 4.12. We've had this known issue in the OSC release notes up until now exactly as is. There isn't anywhere else we could include this procedure at this point, and next release it will be removed completely.